src/components/toolbar/Toolbar.tsx

```diff
@@ -51,17 +53,21 @@ export interface ToolbarProps extends ToolbarBaseProps {
   popoverInteractionKind?: PopoverInteractionType;
 }
 
-export interface ToolbarItemProps
-  extends ToolbarBaseProps,
-    Omit<ButtonProps, 'onClick' | 'tooltipProps'> {
+export interface ToolbarItemProps extends ToolbarBaseProps {
+  id?: string;
   tooltip?: TooltipProps['content'];
   tooltipProps?: Omit<TooltipProps, 'content'>;
+  icon: IconName | ReactElement;
+  active?: boolean;
   onClick?: (
     item: ToolbarItemProps & {
       event: MouseEvent;
     },
   ) => void;
+  className?: string;
   isPopover?: boolean;
+  tag?: ReactNode;
```

Comment:
Can you use `Pick<ButtonProps, ...>` for all the properties that are forwarded to the `Button` component?